ow_lander/src/ow_lander/arm_interface.py

```diff
@@ -171,7 +193,7 @@ def __init__(self):
         self._continue_arm_in_fault = config[
             ArmFaultMonitor.CONTINUE_ARM_IN_FAULT_NAME]
 
-    def _system_faults_callback(self, data):
+    def _system__faults_callback(self, data):
```

Comment:
Is your extra underscore in the wrong place here? I thought you might want it at the beginning to make this method "really private."

Reply:
Oops! This actually is a find-and-replace mistake. Good catch.

Reply:
Fixed